sig-cloud-provider/cloud-provider-extraction-migration/OWNERS

```diff
@@ -2,9 +2,9 @@
 
 reviewers:
   - cheftako
-  - andrewsykim
+  - mcrute
```

Comment:
This directory `cloud-provider-extraction-migration` just contains the OWNERS file. Can we remove this entirely?

Reply:
There's some extra context on this here https://github.com/kubernetes/community/pull/2740#discussion_r225684459, but the TL;DR is that this subproject doesn't own any code (just migrates/extracts a lot of code owned by other SIGs) but it was required for us to have an OWNERs file for each subproject at the time. Would be happy to change this if there's a better alternative :)

Reply:
Thanks for the link, the context is helpful! :)

>  if there's a better alternative

There isn't an alternative yet, needs to be resolved on the steering side: https://github.com/kubernetes/steering/issues/36.